cypress/integration/login.js

```diff
@@ -17,22 +9,45 @@ describe('Sourcebooks login', function() {
     ];
 
     function makeid(length) {
-        var result = '';
-        var characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-        var charactersLength = characters.length;
-        for (var i = 0; i < length; i++) {
-           result += characters.charAt(Math.floor(Math.random() * charactersLength));
-        }
-        return result;
+        return 'Random_Name_' + Math.floor(Math.random() * 1000000000000);

```

Comment:
why you are not using timestamp? It would make it more predictable and event traceable, with random there is still chance...